geode-lucene/src/integrationTest/java/org/apache/geode/cache/lucene/LuceneIndexCreationPersistenceIntegrationTest.java

```diff
@@ -228,7 +228,7 @@ private void verifyQueryResultSize(String indexName, String regionName, String q
       String defaultField, int size) throws Exception {
     LuceneQuery query = luceneService.createLuceneQueryFactory().create(indexName, regionName,
         queryString, defaultField);
-    await().untilAsserted(() -> assertThat(query.findPages()).hasSize(size));
+    await().untilAsserted(() -> assertThat(query.findPages()).toIterable().hasSize(size));
```

Comment:
odd that this needed to change for a minor upgrade.

Reply:
This was changed with assertj 3.12.0.  See https://assertj.github.io/doc/#assertj-core-3-12-0-release-notes.